stories/otp-input/otp-input.stories.tsx

```diff
@@ -88,3 +88,16 @@ export const WithCustomActionButtonSelfHandle: StoryObj<Component> = {
     },
     decorators: [StoryDecorator({ maxWidth: true })],
 };
+
+export const WithPrefix: StoryObj<Component> = {
```

Comment:
can move this story to under WithErrorMessage to match the position in Storybook docs